.github/workflows/build-and-unit-tests.yaml

```diff
@@ -39,7 +39,9 @@ jobs:
     name: ${{ inputs.arch }} ${{ inputs.runner-label }}
     runs-on:
       - ${{ inputs.runner-label }}
-      - "in-service"
+      - cloud-virtual-machine
+      - in-service
+      - config-single-card
```

Comment:
isn't this label redundant?

Reply:
Wondering if it makes sense to have this config-single-card labelled paired with card (ie E150/N300,etc)
Since Raymond mentioned that we should keep arch for t3000 and galaxy since they won't always be wormhole cards, does it make sense to instead use card or product labels?

Examples are because T3Ks also have N300s and Galaxies have N150s 

we also have t3k VMs coming up so putting cloud-virtual-machine and config-t3000 with N300 label could conflict.

imo, it would be more flexible keeping configuration labels for what a machine should have 
VM/BM
Card installed (e150/n150/n300/etc)
configuration (single/t3k/tg/tgg)

Unless, these are set product configurations with set cards, in which case, config-single-card can be removed.

Thoughts?
@TT-billteng @tt-rkim

Reply:
on second thought,
let's table that discussion for another issue, if we want to reorganize labels.
I'll remove it for now since we can use combinations of
VM/BM + card

Reply:
I like ensuring all runners have the 4:

- VM/BM
- card installed
- config (single/t3k/tg/tgg)
- pipeline purpose

However, one wrinkle is that Galaxy doesn't necessarily have a specific card type installed within it. We could use the intermediary driver card as a proxy (ie. current setup uses N150 in the middle). However, this may not always be the case. We should check syseng and @ubcheema to make sure we know what's going on.

Also don't break data pipelines. Check `infra/data_collection/github/utils.py:176` to about 42 lines down.

Reply:
feel like we're overcomplicating this at this point in time

VM/BM
config: E150/N150/N300/T3k/tgg/tgg
pipeline purpose (for special cases)